src/api/app/assets/stylesheets/webui/validations.scss

```diff
@@ -0,0 +1,3 @@
+.error-message {
+    color: var(--bs-danger-text-emphasis);
+}
```

Comment:
How about we use bootstrap with `invalid-feedback` class https://getbootstrap.com/docs/5.3/forms/validation/#server-side

Reply:
But that's what I use https://github.com/openSUSE/open-build-service/pull/16647/files#diff-a6452613bd0bcdcd8c3746c569bad8a787497c8b88eec3e1b4d4037d58325180R4

Reply:
Ah, for the message also... Let me see...

Reply:
Done

Reply:
So we don't need this css?

Reply:
We don't, indeed